lib/subroutine/outputs.rb

```diff
@@ -45,32 +63,49 @@ def output(name, value)
         raise ::Subroutine::Outputs::UnknownOutputError, name
       end
 
-      outputs[name] = value
+      outputs[name] = output_configurations[name].lazy? ? LazyExecutor.new(value) : value
```

Comment:
I believe this will mess with `op_instance.outputs` (which is a hash of all the outputs)

maybe `outputs` can be a method that calls any lazy outputs for backwards compatibility?

Reply:
Thanks updated